app/schemas/project_memoryusage.py

```diff
@@ -0,0 +1,14 @@
+from marshmallow import Schema, fields, validate
+
+
+class ProjectMemoryUsageSchema(Schema):
```

Comment:
I think you need to change this schema to something uniform, because most queries have similar attributes...something like `GraphMetricSchema`, `MetricSchema` ...

Reply:
True that makes sense, let me adjust this

Reply:
This has been adjusted @Mubangizi